recipe/meta.yaml

```
@@ -48,10 +60,9 @@ outputs:
         - test ! -f "${PREFIX}/lib/libucs.a"
         - test -f "${PREFIX}/lib/libuct${SHLIB_EXT}"
         - test ! -f "${PREFIX}/lib/libuct.a"
+        - test -f "${PREFIX}/bin/.ucx-post-link.sh"
         - test -f "${PREFIX}/bin/ucx_info"
-        # Requires driver for GPU test.
-        # Crashes qemu on ppc64le.
-        - ${PREFIX}/bin/ucx_info -v         # [cuda_compiler_version == "None" and not ppc64le]
+        - ucx_info -v
```

> ppc64le fails for some reason, would you mind taking a look? It seems to be failing when running the ucx_info test, but I am not 100% sure given the CI logs were interrupted around that point.

If it fails again, lets just disable it as was the case before

```suggestion
        - ucx_info -v  # [not ppc64le]
```